app/ai/voice/agents/breeze_buddy/observers/manager.py

```
@@ -0,0 +1,169 @@
+
+            for obs, result in zip(eligible, results):
+                if self._action_taken:
+                    return
+                if isinstance(result, Exception):
+                    logger.warning(f"Observer '{obs.name}' check failed: {result}")
+                    continue
+                if result is True:
+                    self._action_taken = True
+                    try:
+                        await obs.execute_action()
+                    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)